### PR TITLE
Update august for offline key access

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Update august for offline key access.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/76577
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/august.markdown [hunk 1]
```diff
@@ -103,6 +103,24 @@ If you have an August Keypad, once you have enabled the August component, you sh
 
 - Keypad Battery
 
+## Integration with Yale Access Bluetooth
+
+Following Assa Abloy, Yale's parent company, purchasing August in 2017, most newer devices use the Yale Access branding. 
+
+The [Yale Access Bluetooth](/integrations/yalexe_ble) provides local control over Bluetooth of many Yale Access locks and some August locks that use the same system. 
+
+For locks that support the Yale Access system, the august integration can keep your offline access keys up to date to ensure you can operate your lock over Bluetooth.  The following requirements must be met for the offline key updates to work:
```
Comment (bdraco) [inline source/_integrations/august.markdown]:
```suggestion
For locks that support the Yale Access system, the August integration can keep your offline access keys up to date to ensure you can operate your lock over Bluetooth.  The following requirements must be met for the offline key updates to work:
```

File: source/_integrations/august.markdown [hunk 2]
```diff
@@ -103,6 +103,24 @@ If you have an August Keypad, once you have enabled the August component, you sh
...
+For locks that support the Yale Access system, the August integration can keep your offline access keys up to date to ensure you can operate your lock over Bluetooth.  The following requirements must be met for the offline key updates to work:
+
+- The August integration must support the lock
+- The [Yale Access Bluetooth integration](/integrations/yalexe_ble) must support the lock
+- The Bluetooth integration must be active and functional
+- The account logged in with the August integration must have the offline keys.
+
+### Troubleshooting offline keys updates
+
+- If you do not know which account has the offline keys, configure august integration with each different Owner account.
```
Comment (bdraco) [inline source/_integrations/august.markdown]:
```suggestion
- If you do not know which account has the offline keys, configure August integration with each different Owner account until you find the one that holds the keys. You may need to make a new owner account and grant the account access to your lock to force the keys to synchronize with the cloud service.
```

File: source/_integrations/august.markdown [hunk 3]
```diff
@@ -103,6 +103,24 @@ If you have an August Keypad, once you have enabled the August component, you sh
...
+
+Following Assa Abloy, Yale's parent company, purchasing August in 2017, most newer devices use the Yale Access branding. 
+
+The [Yale Access Bluetooth](/integrations/yalexe_ble) provides local control over Bluetooth of many Yale Access locks and some August locks that use the same system. 
+
+For locks that support the Yale Access system, the August integration can keep your offline access keys up to date to ensure you can operate your lock over Bluetooth.  The following requirements must be met for the offline key updates to work:
+
+- The August integration must support the lock
+- The [Yale Access Bluetooth integration](/integrations/yalexe_ble) must support the lock
+- The Bluetooth integration must be active and functional
```
Comment (bdraco) [inline source/_integrations/august.markdown]:
```suggestion
- The Bluetooth integration must be active and functional
- The lock must be discoverable by the [Yale Access Bluetooth integration](/integrations/yalexe_ble)
```

File: source/_integrations/august.markdown [hunk 4]
```diff
@@ -103,6 +103,25 @@ If you have an August Keypad, once you have enabled the August component, you sh
 
 - Keypad Battery
 
+## Integration with Yale Access Bluetooth
+
+Following Assa Abloy, Yale's parent company, purchasing August in 2017, most newer devices use the Yale Access branding. 
+
+The [Yale Access Bluetooth](/integrations/yalexe_ble) provides local control over Bluetooth of many Yale Access locks and some August locks that use the same system. 
+
+For locks that support the Yale Access system, the August integration can keep your offline access keys up to date to ensure you can operate your lock over Bluetooth.  The following requirements must be met for the offline key updates to work:
```
Comment (frenck) [inline source/_integrations/august.markdown]:
```suggestion
For locks that support the Yale Access system, the August integration can keep your offline access keys up to date to ensure you can operate your lock over Bluetooth. The following requirements must be met for the offline key updates to work:
```

File: source/_integrations/august.markdown [hunk 5]
```diff
@@ -103,6 +103,25 @@ If you have an August Keypad, once you have enabled the August component, you sh
...
+
+- The August integration must support the lock
+- The [Yale Access Bluetooth integration](/integrations/yalexe_ble) must support the lock
+- The Bluetooth integration must be active and functional
+- The lock must be discoverable by the [Yale Access Bluetooth integration](/integrations/yalexe_ble)
+- The account logged in with the August integration must have the offline keys.
```
Comment (frenck) [inline source/_integrations/august.markdown]:
```suggestion
- The August integration must support the lock.
- The [Yale Access Bluetooth integration](/integrations/yalexe_ble) must support the lock.
- The Bluetooth integration must be active and functional.
- The lock must be discoverable by the [Yale Access Bluetooth integration](/integrations/yalexe_ble).
- The account logged in with the August integration must have the offline keys.
```

File: source/_integrations/august.markdown [hunk 6]
```diff
@@ -103,6 +103,25 @@ If you have an August Keypad, once you have enabled the August component, you sh
...
+
+### Troubleshooting offline keys updates
+
+- If you do not know which account has the offline keys, configure August integration with each different Owner account until you find the one that holds the keys. You may need to make a new owner account and grant the account access to your lock to force the keys to synchronize with the cloud service.
+- Ensure the lock is in range and discoverable by the [Yale Access Bluetooth integration](/integrations/yalexe_ble)
```
Comment (frenck) [inline source/_integrations/august.markdown]:
```suggestion
- Ensure the lock is in range and discoverable by the [Yale Access Bluetooth integration](/integrations/yalexe_ble).
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by bdraco** [comment]
Thanks

**Comment by bdraco** [comment]
When I get all the old august locks with with the bluetooth integration I'll do some supported brands work to try to make this a bit easier to understand.

Unfortunately Yale's branding is so convoluted, and they seem to have so much brand equity in August, that they can't make up their mind on what to call things.

**Pull request merged**